contrib/bvxm/algo/pybvxm_algo.cxx

```diff
@@ -0,0 +1,81 @@
+#include "pybvxm_algo.h"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/numpy.h>
+
+#include <bvxm/bvxm_voxel_world.h>
+#include <bvxm/bvxm_world_params.h>
+#include <bvxm/algo/bvxm_create_scene_xml.h>
+#include <vul/vul_file.h>
+
+namespace py = pybind11;
+
+namespace pyvxl { namespace bvxm { namespace algo {
+
+bvxm_voxel_world create_voxel_world(std::string const& vox_dir, std::string const& lvcs_path,
```

Comment:
Indent

Reply:
I'm of the opinion that namespaces shouldn't be indented. It unnecessarily nests all of your code. If I remove all the indentation from the codebase to make it consistent would that be acceptable?